src/components/Users/Users.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getLatestUsers } from '../../actions/usersActions';
+import './Users.scss';
+import PersonPlaceholder from '../../assets/images/person.png';
+import Button from '../Button/Button';
+import ContentLoader from '../Helpers/ContentLoader';
+
+export class Users extends Component {
+  componentDidMount() {
+    const { onGetLatestUsers } = this.props;
+    onGetLatestUsers();
+  }
+
+  render() {
+    const {
+      users,
+      history: { push },
+      loading,
+    } = this.props;
+    return (
+      <div className="latest-users">
+        {loading
+        && [...Array(5).keys()].map((value) => (
+          <ContentLoader key={value} />
+        ))}
```

Comment:
Can't you make the `ContentLoader` component with custom numbers instead of everything hard coding Array(5)?

Reply:
Alright